### PR TITLE
Limit the flip delay in the other direction to try to work around #10763.

### PR DESCRIPTION
**Pull request by hrydgard**
This is getting extremely ugly...

### CODE REVIEW

**Comment by unknownbrackets** [comment]
Yeah... well, at least it doesn't appear to be incorrectly triggering in Jeanne d'Arc, Final Fantasy 4, or Tales of Phantasia.

-[Unknown]

**Comment by zminhquanz** [comment]
It nearly finish , i think it's ready for release 1.6 version

File: Core/HLE/sceDisplay.cpp [hunk 1]
```diff
@@ -713,6 +714,7 @@ void __DisplayFlip(int cyclesLate) {
 			// Check first though, might've just quit / been paused.
 			if (coreState == CORE_RUNNING) {
 				coreState = CORE_NEXTFRAME;
+				DEBUG_LOG(SCEDISPLAY, "No recent flip - displaying anyway.");
```
Comment (unknownbrackets) [inline Core/HLE/sceDisplay.cpp]: nit: This gets logged even for normal flips right?

-[Unknown]

**Comment by hrydgard** [comment]
I can't find any bad effects, let's go for it.

**Comment by hrydgard** [comment]
I'm gonna revert this and see if I can come up with something else...

**Pull request merged**